Requirements for ICMP tests on Private Locations (#28163)
DataDog/documentation: content/en/synthetics/troubleshooting/_index.md

Confirm you are not seeing [out of memory issues][102] with your private location deployments. If you have tried scaling your workers instances following the [dimensioning guidelines][103] already, reach out to [Datadog Support][104].

### Requirements for ICMP tests running on private location

> ICMP tests use the `ping` command to assess network routes and connectivity to a host. `ping` opens a raw socket through which ICMP packets are sent/received and so `ping` requires the `NET_RAW` capability to allow for the creation of raw sockets. If your container is configured with a security context that drops or removes this capability, ICMP tests will not be able to function properly on the private location.

Reviewer: ```suggestion
ICMP tests use the `ping` command to assess network routes and connectivity to a host. `ping` opens a raw socket to send ICMP packets through, so it requires the `NET_RAW` capability to allow for the creation of raw sockets. If your container is configured with a security context that drops or removes this capability, ICMP tests will not be able to function properly on the private location.
```

> Additionally, `ping` requires elevated privileges in order to create the raw socket. The private location cannot execute ICMP tests if the private location is configured with a security context that restricts elevated privileges.

Reviewer: ```suggestion
Additionally, `ping` requires elevated privileges to create the raw socket. The private location cannot execute ICMP tests if the private location is configured with a security context that restricts elevated privileges.
```
